app/assets/javascripts/components/student/student_page.es6.jsx

```diff
@@ -43,8 +44,9 @@ class StudentPage extends Component {
     this.setState({ sidebar: !this.state.sidebar });
   }
 
-  renderOverlay(event) {
-    this.setState({ overlay: true });
+  renderOverlay(card_type) {
+    this.setState({ overlay: true,
+                    type: card_type });
```

Comment:
No need for the extra space here, and since the state is called `type` we might as well just call `card_type` `type` as well.